pkg/ipam/poolallocator/allocator.go

```diff
@@ -252,6 +250,33 @@ func (a *IPPoolAllocator) removeIPAddressState(ipPool *v1alpha2.IPPool, ip net.I
 
 }
 
+func (a *IPPoolAllocator) getPreviousAllocation(podOwner *v1alpha2.PodOwner) (net.IP, *v1alpha2.SubnetInfo, error) {
```

Comment:
Do you mean `getExistingAllocationFromPodOwner`? Maybe we need to add some comments.

Reply:
getExistingAllocation sounds better to me. Changed to that.